src/world/footpath.c

```diff
@@ -1472,16 +1472,25 @@ void sub_6A759F()
  */
 static void footpath_unown(int x, int y, rct_map_element *pathElement)
 {
-	int ownershipUnk = 0;
-	int z = pathElement->base_height;
 	rct_map_element *surfaceElement = map_get_surface_element_at(x >> 5, y >> 5);
```

Comment:
Might as well make this `map_get_surface_element_at(x / 32, y / 32)`. Seems more readable to me than `>> 5`